### PR TITLE
python example compliant with python v3.x

### PR DESCRIPTION
**Pull request by dnascimb**
Pretty basic. If you feel there is a better way to incorporate python v3.x support, feel free. Just trying to contribute as I ran into the issues marked in the file when making use of the example code.

### CODE REVIEW

File: python3_example.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    self.access_filters = json.dumps(access_filters)
+
+
+class URL:
+  def __init__(self, looker, user, session_length, embed_url, force_logout_login=False):
+    self.looker = looker
+    self.user = user
+    #python2 version
+    #self.path = '/login/embed/' + urllib.quote_plus(embed_url)
+    self.path = '/login/embed/' + urllib.parse.quote_plus(embed_url)
```
Comment (dnascimb) [inline python3_example.py]: here ->

File: python3_example.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    self.session_length = json.dumps(session_length)
+    self.force_logout_login = json.dumps(force_logout_login)
+
+  def set_time(self):
+    self.time = json.dumps(int(time.time()))
+
+  def set_nonce(self):
+    #python2
+    #self.nonce = json.dumps(binascii.hexlify(os.urandom(16)))
+    self.nonce = json.dumps(binascii.hexlify(os.urandom(16)).decode('ascii'))
```
Comment (dnascimb) [inline python3_example.py]: here ->

File: python3_example.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    string_to_sign = string_to_sign + self.session_length        + "\n"
+    string_to_sign = string_to_sign + self.user.external_user_id + "\n"
+    string_to_sign = string_to_sign + self.user.permissions      + "\n"
+    string_to_sign = string_to_sign + self.user.models           + "\n"
+    string_to_sign = string_to_sign + self.user.access_filters
+
+    #python2
+    #signer = hmac.new(self.looker.secret, string_to_sign.encode('utf8'), sha1)
+    signer = hmac.new(self.looker.secret.encode('utf-8'), string_to_sign.encode('utf8'), sha1)
+    self.signature = base64.b64encode(signer.digest().rstrip('\n'))
```
Comment (dnascimb) [inline python3_example.py]: here ->

File: python3_example.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+              'models':              self.user.models,
+              'access_filters':      self.user.access_filters,
+              'signature':           self.signature,
+              'first_name':          self.user.first_name,
+              'last_name':           self.user.last_name,
+              'force_logout_login':  self.force_logout_login}
+
+    #python2
+    #query_string = '&'.join(["%s=%s" % (key, urllib.quote_plus(val)) for key, val in params.iteritems()])
+    query_string = '&'.join(["%s=%s" % (key, urllib.parse.quote_plus(val)) for key, val in params.items()])
```
Comment (dnascimb) [inline python3_example.py]: here ->

**Comment by ohimors** [comment]
Why is this PR still open?

**Comment by jkaster** [comment]
Thanks for your work on this. There is a new version of the python example committed to master now that works in both 2.7 and 3.6. Please let us know if you have any trouble with it.

**Pull request closed without merging**